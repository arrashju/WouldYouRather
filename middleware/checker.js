const checker = (store) => (next) => (action) => {
  return next(action);
};

export default checker;
