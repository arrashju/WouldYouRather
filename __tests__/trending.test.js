import { render } from "@testing-library/react";
import { Provider as ReduxProvider } from "react-redux";
import reducer from "../reducers";
import middleware from "../middleware";
import { createStore } from "redux";
import { composeWithDevTools } from "redux-devtools-extension";
import Trending from "../pages/trending";

const store = createStore(reducer, composeWithDevTools(middleware));

describe("Renders trending page", () => {
  it("trending render before loading elements should match snapshot", () => {
    const { container } = render(
      <ReduxProvider store={store}>
        <Trending />
      </ReduxProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
