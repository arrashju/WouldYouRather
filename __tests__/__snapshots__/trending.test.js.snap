// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders trending page trending render before loading elements should match snapshot 1`] = `
<div>
  <div
    class="sc-kgflAQ daQuIQ"
    hidden=""
  >
    <div
      class="sc-hHLeRK"
    />
  </div>
  <div
    class="sc-bBrHrO iMXppO"
  >
    <div
      class="sc-ivTmOn ilVgBw"
    >
      <button
        class="sc-cxabCf hDtBGU"
      >
        All
      </button>
      <button
        class="sc-cxabCf hDtBGU"
      >
        Sports
      </button>
      <button
        class="sc-cxabCf hDtBGU"
      >
        Business
      </button>
      <button
        class="sc-cxabCf hDtBGU"
      >
        Answered
      </button>
      <button
        class="sc-cxabCf eLiLvp"
      >
        Unanswered
      </button>
    </div>
    <small
      class="sc-fLlhyt ecEZkc"
    >
      0
       Polls
    </small>
    <div
      class="sc-llJcti ksgYUU"
    >
      Trending Polls
    </div>
    <div
      class="sc-iIPllB jocqib"
    >
      Unanswered
    </div>
    <div
      class="sc-kLLXSd hDIBJC"
    />
  </div>
</div>
`;
